### PR TITLE
Link to oidc spec for claims_supported

### PR DESCRIPTION
**Pull request by michielbdejong**
As discussed in https://gitter.im/solid/test-suite?at=6193a7de7db1e3753ea224f3

### CODE REVIEW

File: data/solid-oidc.jsonld [hunk 1]
```diff
@@ -97,7 +97,7 @@
     "comment": "The server metadata MUST indicate support for the webid claim in the claims_supported field.",
     "requirementSubject": "OpenIDProvider",
     "requirementLevel": "MUST",
-    "requirementReference": "https://solid.github.io/solid-oidc/#discovery" },
+    "requirementReference": "https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata" },
```
Comment (acoburn) [inline data/solid-oidc.jsonld:100]: We should be linking to Solid-OIDC here, not OpenID Connect

**Review by acoburn — CHANGES_REQUESTED** [review]

File: data/solid-oidc.jsonld (hunk 1 above)
Comment (michielbdejong) [reply data/solid-oidc.jsonld:100]: But https://solid.github.io/solid-oidc/#discovery says nothing about `claims_supported`

Comment (michielbdejong) [reply data/solid-oidc.jsonld:100]: So maybe that is the root of the problem :)

**Comment by michielbdejong** [comment]
OK so let's not merge this, but just update the solid-oidc spec instead:
https://github.com/solid/solid-oidc/issues/56

**Pull request closed without merging**